cves/kernel/CVE-2016-9754.yml

```diff
@@ -106,17 +108,17 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 7a8e76a3829f1067b70f715771ff88baf2fbf3c3
-  note: Discovered automatically by archeogit.
+  note: 'Discovered automatically by archeogit. Introduces the ring buffer.'
 - commit: 83f40318dab00e3298a1f6d0b12ac025e84e478d
-  note: Discovered automatically by archeogit.
+  note: 'Discovered automatically by archeogit. Adds the capability to remove pages from a ring buffer without destroying any existing data in it.'
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 3
```

Comment:
Overall great! Your explanations are clear and easy to understand. The minor errors are consistency in capitalization and punctuation across the document.
I would give this a 5.